manifest.yaml

```diff
@@ -12,9 +12,24 @@ dotnet:
         image_version: 8.0.11
         image_type: bookworm-slim
         env_version_key: DOTNET_VERSION
+    '9.0/aspnet':
+      data:
+        image_repo: mcr.microsoft.com/dotnet/runtime
```

Comment:
should this be `aspnet` rather than `runtime`?

Reply:
oops copied and pasted wrong block. fixed